nvm.sh

```diff
@@ -681,9 +681,10 @@ nvm_ls_remote_iojs() {
     PATTERN=".*"
   fi
   VERSIONS="$(nvm_download -L -s $NVM_IOJS_ORG_VERSION_LISTING -o - \
-    | command sed 1d \
-    | command sed "s/^/$(nvm_iojs_prefix)-/" \
-    | command cut -f1 \
+    | command sed "
+        1d;
+        s/^/$(nvm_iojs_prefix)-/;
+        s/\s.*//" \
```

Comment:
Those `sed` differences between Mac and Linux lol. `[[:blank:]]` works on both, let's try again...